classes/submissionFile/SubmissionFile.inc.php

```diff
@@ -374,10 +362,40 @@ public function setChapterId($chapterId)
     {
         $this->setData('chapterId', $chapterId);
     }
+
+    public function itsOnReviewFileStage(): bool
```

Comment:
This is stylistic, but I'd rather just do the comparison in the calling code rather than code up one-liners for each type of file. If the file type set changes, it's easy to forget to adjust these too, and it's harder to identify affected code when searching for e.g. a constant if you have to remember to also search for this function by name.

Reply:
There's three ideas behind this implementation:

1) The DRY principle, to avoid the knowledge duplication (more [here](https://thevaluable.dev/dry-principle-cost-benefit-example/)) across the codebase;
2) The PRS-12 "rules" about lines ([2.3 Lines section](https://www.php-fig.org/psr/psr-12/)), where lines shouldn't be longer than 80 characters (as I have found at some places with "inline validation/comparison"); and
3) Avoid an Anemic Domain Model (more (here)[https://martinfowler.com/bliki/AnemicDomainModel.html]).

Reply:
I think I'd still rather remove these, at least for now, and consider their addition more broadly outside of this PR. It's easy to partially introduce new conventions and never make them universal, which causes a lot of confusion.

Reply:
I will revert the changes where I used them and create an issue to enable this in a near future.